src/OpenSpaceToolkit/Astrodynamics/Access/Generator.cpp

```diff
@@ -840,7 +840,7 @@ Array<physics::time::Interval> Generator::computePreciseCrossings(
         Instant intervalStart = anAnalysisInterval.getStart();
 
         // compute start crossing if it is not the start of the requested analysis interval
```

Comment:
```suggestion
        // Compute start crossing if it minus one step is after the analysis interval lower bound
```

Reply:
Will update it but I think we can make it a bit clearer.